deploy/local_single_user_deploy.sh

```diff
@@ -536,17 +461,17 @@ parse_args() {
                 NUM_RECORDING_GROUPS="$2"
                 shift 2 ;;
             -w|--work-dir)
```

Comment:
mismatch with help page. help page says `-w|--install-dir` while it says `-w|--work-dir` here.

Reply:
I am not sure if it is caused by this mismatching. But `./local_single_user_deploy.sh -i -l` indefinitely prints `realpath: '': No such file or directory`.